src/hotspot/share/gc/serial/cardTableRS.cpp

```diff
@@ -155,9 +155,9 @@ void CardTableRS::invalidate_or_clear(Generation* old_gen) {
   MemRegion used_mr = old_gen->used_region();
   MemRegion to_be_cleared_mr = old_gen->prev_used_region().minus(used_mr);
   if (!to_be_cleared_mr.is_empty()) {
-    clear(to_be_cleared_mr);
+    clear_MemRegion(to_be_cleared_mr);
   }
-  invalidate(used_mr);
+  dirty_MemRegion(used_mr);
```

Comment:
Now that `invalidate()` isn't called any more, it might be useful to rename the method to `dirty_or_clear()` or something. In general I think that "dirty" is more specific and clear than "invalidate", but the "invalidate" term probably stems from wanting to be generic (not only here).
So feel free to ignore this suggestions.

Reply:
`clear_into_younger` and `invalidate_or_clear/dirty_or_clear` are not very descriptive. I will refactor them in another PR.